.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -9,12 +9,12 @@
 1. - [ ] Retrieve the final run of the [Stage-DotNet](https://dev.azure.com/dnceng/internal/_build?definitionId=792&_a=summary) pipeline from internal release communications.
 1. - [ ] Run the [`source-build-release-official`](https://dev.azure.com/dnceng/internal/_build?definitionId=1229) pipeline.
      - When staging the pipeline run, click "Resources" and select the final run of `Stage-DotNet` mentioned above.
-     - If a special source-build tag was created for the release, check `Use custom tag` and set the `Custom tag` parameter. Otherwise, a release name from the release manifest is used.
+     - If a special source-build tag was created for the release, check `Use custom tag` and set the `Custom release tag` parameter. Otherwise, a release name from the release manifest is used by default.
+     - In case a different installer commit is being released than the one in the associated staging pipeline, check `Use specific pipeline run IDs` and input the official build numbers of the builds belonging to the released commit for the parameter(s) relevant to the target .NET version ("Specific {pipeline} run name").
+     - In case you will be using a gist to craft the GitHub announcement, create a secret gist with a single `.md` file. Name the gist what you want the title of the announcement to be, e.g. `.NET 8 March 2023 Update - .NET 8.0.0-preview.2.23128.3 / SDK 8.0.100-preview.2.23153.6`. The title of the gist file is not relevant (make it an `.md` though). Suggested content of the announcement will be shown in the `Pre-Release` stage.
```

Comment:
I am adding more explicit difference between gist title and file name:
```suggestion
     - In case you will be using a gist to craft the GitHub announcement, create a secret gist with a single `.md` file. Name the whole gist what you want the title of the announcement to be, e.g. `.NET 8 March 2023 Update - .NET 8.0.0-preview.2.23128.3 / SDK 8.0.100-preview.2.23153.6`. The name of the gist file is not relevant (make it an `.md` though). Suggested content of the announcement will be shown in the `Pre-Release` stage.
```